source_files/gpu.h

```diff
@@ -30,6 +30,7 @@ typedef struct{
       int verbose;      //verbosity
       int loadingdone;  //variable to use if the initialization of GPUs are done
       int init_count;   // Counts devices that already initialized and probably started the workload 
+      unsigned long long *loadvar; //provides termination information
```

Comment:
In a next iteration, we should just use the global variabe LOADVAR instead of having pointers to it in both the watchdog and the gpu struct. Or we remove the global variable in the watchdog.
Currently, I can live with it, but it hurts having both concurrently.